Makefile

```diff
@@ -1,4 +1,5 @@
 HUB ?= quay.io/maistra-dev
+SHA ?= $(shell git rev-parse --short=8 HEAD)
```

Comment:
istio uses date + branch: https://github.com/istio/tools/blob/master/docker/build-tools/build-and-push.sh#L29
can we use the same?

Reply:
we can add date, but I'm not sure about branch - we keep everything on the `main` right now